roofit/roofitcore/inc/RooFit/TestStatistics/RooAbsL.h

```diff
@@ -76,6 +74,14 @@ class RooAbsL {
          }
       }
 
+      bool operator==(const Section& rhs) {
+         if (begin_fraction == rhs.begin_fraction && end_fraction == rhs.end_fraction) {
+            return true;
+         } else {
+            return false;
+         }
```

Comment:
```suggestion
         return begin_fraction == rhs.begin_fraction && end_fraction == rhs.end_fraction;
```
Maybe do this to save a few lines?